spec/controllers/ops_controller/settings/upload_spec.rb

```diff
@@ -3,6 +3,166 @@
     stub_user(:features => :all)
   end
 
+  describe '#upload_png' do
+    let(:dir) { File.dirname(__FILE__) }
```

Comment:
This simplified to just
```suggestion
    let(:dir) { __dir__ }
```

but then it's small enough you don't need the let and you can just use it directly with File.join. (e.g. `File.join(__dir__, "data/test.png")`